recipes/pycompilation/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pycompilation" %}
+{% set version = "0.4.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycompilation-{{ version }}.tar.gz
+  sha256: fa8095aa956055b37de965d97bec0ba1a29d8b7354cc73bb4094d7286d8b883b
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win or osx]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  files:
+    - on_the_fly_main.py
+  requires:
+    - {{ compiler('c') }}
+    - cython
+    - numpy
+    - pip
+  imports:
+    - pycompilation
+  commands:
+    - pip check
+    - python on_the_fly_main.py
+
+about:
+  home: https://github.com/bjodah/pycompilation
+  summary: pycompilation is a package for meta programming. It aims to support
```

Comment:
Seems like something is missing here?

Reply:
It is a copy from pypi:

![image](https://github.com/conda-forge/staged-recipes/assets/276007/f206cc30-2864-410e-a513-216c48883a02)

Reply:
Sure, it's a bug in the setup.py that extracts it from here https://github.com/bjodah/pycompilation/blob/master/pycompilation/__init__.py#L2

I think we should still try to have a meaningful description.